staging/src/k8s.io/apiserver/pkg/server/options/etcd.go

```diff
@@ -66,9 +69,10 @@ type EtcdOptions struct {
 	transformerOverrides   map[schema.GroupResource]value.Transformer
```

Comment:
`LoadEncryptionConfig` returns the map which we then convert to Dynamic transformers. So don't we need a map var to store that initial transformer overrides?

So if we convert this to a dynamic transformer it will be like a chicken and egg situation. We want to convert the map to dynamic transformers, but now we can't do it since dynamic transformers need dynamic transformers to initialize.

Reply:
nevermind. removed it.